m4/ac_path_kernel_source.m4

```diff
@@ -135,6 +135,10 @@ AC_DEFUN([AC_KERNEL_CHECKS],
     AC_DEFINE([HAVE_LATEST_APPLY_TO_PAGE_RANGE], 1, [Have latest page iterator])
   ], [])
 
+  AC_CHECK_DECLS([vm_flags_set], [
+    AC_DEFINE([HAVE_VM_FLAGS_SET_FUNC], 1, [Have vm_flags_set function])
```

Comment:
AC_CHECK_DECLS should already define HAVE_DECL_VM_FLAGS_SET_FUNC

Reply:
fixed, build tested on 3.10 and 6.3+